modules/configuration/src/main/java/org/apache/ignite/configuration/ConfigurationChanger.java

```diff
@@ -176,7 +200,7 @@ public void initialize(Class<? extends ConfigurationStorage> storageType) {
             throw new ConfigurationValidationException(validationIssues);
 
         try {
-            change(defaultsNode, storageType).get();
+            change0(defaultsNode, storageInstances.get(storageType)).get();
```

Comment:
It's interesting that you've decided to remove the method with the name `change` and not `change0` =)

Reply:
I'll rename it in the future)

Reply:
why not now?

Reply:
Ok